doc/languages-frameworks/javascript.section.md

```diff
@@ -204,6 +204,7 @@ buildNpmPackage rec {
 * `npmBuildFlags`: Flags to pass to `npm run ${npmBuildScript}`.
 * `npmPackFlags`: Flags to pass to `npm pack`.
 * `npmPruneFlags`: Flags to pass to `npm prune`. Defaults to the value of `npmInstallFlags`.
+* `makeWrapperArgs`: Flags to pass to `makeWrapper`, added to the node wrapper calling the generated `.js` scripts.
```

Comment:
Maybe make it clearer this is only used for `bin`s defined in package.json?

```suggestion
* `makeWrapperArgs`: Flags to pass to `makeWrapper` when wrapping package-defined `bin` entrypoints with `node`.
```

I'm open to wording suggestions, since it's a bit early this morning for me to write scrutable documentation, but this is just a suggestion of what I'm thinking